rust/operator-binary/src/controller.rs

```diff
@@ -956,16 +959,41 @@ fn build_opa_start_command(merged_config: &OpaConfig, container_name: &str) -> S
             level: Some(log_level),
         }) = log_config.console
         {
-            console_logging_off = log_level == LogLevel::NONE
+            console_logging_enabled = log_level != LogLevel::NONE
         }
     }
 
+    let decision_logs_to_stdout = merged_config.decision_logging.console;
+    let decision_logs_to_file = merged_config.decision_logging.file;
+
+    // Redirects matter!
+    // We need to watch out, that the following "$!" call returns the PID of the main (opa-bundle-builder) process,
+    // and not some utility (e.g. multilog or tee) process.
+    // See https://stackoverflow.com/a/8048493
+
+    // *Technically*, we would also need to calculate `file_logging_enabled` and take that into account. However, this
+    // is a very nice edge-case and would make things more complicated and error-prone.
+    let logging_redirects = if console_logging_enabled {
+        match (decision_logs_to_file, decision_logs_to_stdout) {
+            (true, true) => format!(" &> >(tee >(/stackable/multilog s{OPA_ROLLING_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name}))"),
```

Comment:
Can we please extract the `multilog s{OPA_ROLLING_LOG_FILE_SIZE_BYTES} n{OPA_ROLLING_LOG_FILES} {STACKABLE_LOG_DIR}/{container_name}` part into a `const`?